tslint.json

```diff
@@ -0,0 +1,63 @@
+{
+  "rules": {
+    "class-name": true,
+    "comment-format": [true, "check-lowercase", "check-space"],
+    "curly": true,
+    "eofline": true,
+    "forin": true,
```

Comment:
There's a practice of omitting the `if` statement because of [performance reasons](https://jsperf.com/hasownproperty-vs-in-vs-undefined/12). I'm not sure if we're not getting to opinionated with this one.